protos/publisher_list.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option optimize_for = LITE_RUNTIME;
+
+package publishers_pb;
+
+message PublisherList {
```

Comment:
Please use the current version of the proto definition file [here](https://docs.google.com/document/d/19zp-GSqPzp6mqFBcy482RUrVM0gEi7HySxSWlmM1Vmg/edit). Note that `publisher_type` was removed (it can be added in a future version if we have a use for it).

Also, note that `BROTLI_COMPRESSION` is not supported yet. Let's get raw encoding working first. Also, I have another compression technique I'm working on that will compress the list better than Brotli and much faster.